IPs/RSKIP177.md

```diff
@@ -0,0 +1,41 @@
+# Merged Mining Capabilities Extension
+
+|RSKIP          |177           |
+| :------------ |:-------------|
+|**Title**      |Merged Mining Capabilities Extension |
+|**Created**    |APR-2020 |
+|**Author**     |MP |
+|**Purpose**    |Sca |
+|**Layer**      |Node |
+|**Complexity** |1 |
+|**Status**     |Adopted |
+
+## Abstract
+
+The present RSKIP introduces an extension of merged mining capabilities. It allows summarized data to be hashed as part of the merged mining hash of the block. For this to be possible, network consensus must be changed.
+
+## Motivation
+
+The ability to extend the merged mining process to include information other than the block’s hash in the merged mining hash offers a myriad of new applications to the miner’s hashing power without affecting their performance.
+
+Potential applications may involve new mechanisms for other blockchains to do merged mining with RSK and by transitivity with BTC increasing their level of security.  Moreover, any kind of summarized information can be added to the merged mining hash (if conditions apply) and be validated with PoW security.
```

Comment:
"by transitivity with BTC increasing their level of security" unclear.
Remove or rephrase.